### PR TITLE
API/Resources: Add examples for retrieving API Keys & updating their expiry date

### PR DESCRIPTION
**Pull request by thgreasi**
Change-type: minor
See: https://jel.ly.fish/support-thread-1-0-0-front-cnv-essp2od

### CODE REVIEW

File: config/dictionaries/resource.json [hunk 1]
```diff
@@ -217,6 +217,47 @@
...
+        "id": "get-api-keys-by-device-uuid",
+        "summary": "Get all API keys of a device by UUID",
+        "description": "",
+        "method": "GET",
+        "endpoint": "/v6/api_key",
+        "filters": "?\\$filter=is_of__actor/any(a:a/is_of__device/any(d:d/uuid%20eq%20'<DEVICE_UUID>'))"
+      },
+      {
+        "id": "get-api-keys-by-application-slug",
+        "summary": "Get all API keys of a application by slug",
```
Comment (klutchell) [inline config/dictionaries/resource.json:230]:
```suggestion
        "summary": "Get all API keys of an application by slug",
```

File: config/dictionaries/resource.json [hunk 2]
```diff
@@ -217,6 +217,47 @@
...
+        "description": "",
+        "method": "GET",
+        "endpoint": "/v6/api_key",
+        "filters": "?\\$filter=is_of__actor/any(a:a/is_of__application/any(ioa:ioa/slug%20eq%20'<SLUG>'))"
+      },
+      {
+        "id": "get-api-keys-by-user",
+        "summary": "Get all user API keys of the authenticated user",
```
Comment (klutchell) [inline config/dictionaries/resource.json:238]:
```suggestion
        "summary": "Get all user API keys for the authenticated user",
```

File: config/dictionaries/resource.json [hunk 3]
```diff
@@ -217,6 +217,47 @@
...
+        "description": "",
+        "method": "GET",
+        "endpoint": "/v6/api_key",
+        "filters": "?\\$filter=is_of__actor/any(a:a/is_of__user%20eq%20<USER_ID>))"
+      },
+      {
+        "id": "set-api-key-expiry-date",
+        "summary": "Update the expiry date to an API Key",
```
Comment (klutchell) [inline config/dictionaries/resource.json:246]:
```suggestion
        "summary": "Update the expiry date for an API Key",
```

File: config/dictionaries/resource.json [hunk 4]
```diff
@@ -217,6 +217,47 @@
...
+        "description": "Note: setting a past date as the expiry, can be used to expire an API Key immediately.",
```
Comment (klutchell) [inline config/dictionaries/resource.json:247]:
```suggestion
        "description": "Note: Select a date in the past to expire an API Key immediately.",
```

**Comment by thgreasi** [comment]
4 commands, 4 review comments :facepalm:

**Comment by thgreasi** [comment]
I self-certify!

**Review by klutchell — APPROVED** [review]

**Pull request merged**